src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fix issue `match-results-forward` command generates Sarif missing `VersionControlDetails` data. [#2487](https://github.com/microsoft/sarif-sdk/pull/2487)
```

Comment:
>Fix issue [](http://example.com/codeflow?start=10&length=9)

Please use this:

'Resolve issue where `match-results-forward` command fails to generate `VersionControlDetails` data.